pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -62,7 +64,8 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileKnativeServing{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	clientConfig, _ := clientcmd.BuildConfigFromFlags("", "")
```

Comment:
What are these flags for?

Reply:
Was trying to use a config passed from the manager, but created it here for a quick test.
should be masterurl and kubeconfig passed into.

Reply:
Check https://github.com/knative/serving-operator/pull/78 for a better way to access the cfg. May use that one instead.